src/main/webapp/styles/homestyle.css

```diff
@@ -0,0 +1,48 @@
+/* Navigation adjustments for home page */
+#header {
+  clear: both;
```

Comment:
We will have to do this on every page because the nav bar is on every page so maybe we could add a header to every page and put this in general.css?

Reply:
Good suggestion, updated all the styled pages so they all have a consistent header.